src/store/quotes.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const quotes = createSlice({
+  name: 'quotes',
+  initialState: {
+    availableQuotes: [],
+    quoteIndex: 0
+  },
+  reducers: {
+    nextQuote(state) {
+      const hi = state.availableQuotes.length - 1;
+      const availableQuoteIndex = Math.floor(Math.random() * hi);
+
+      [state.quoteIndex] = state.availableQuotes.splice(availableQuoteIndex, 1);
+    },
+    resetQuotes(state, action) {
+      state.availableQuotes = new Array(action.payload)
```

Comment:
```suggestion
      state.availableQuotes = []
```
Is this not possible?

Reply:
I'm trying to make an array of indices counting from 0 to action.payload, I reckon `[].fill(0)` would just result in `[]`.